docs/getting-started.rst

```diff
@@ -161,74 +158,45 @@ Columns returned by the simulator
-+------------------------+-----------------------------------------------------+-------+
-|alg2_kdu                |Housing cost covered by social assistance            | float |
-+------------------------+-----------------------------------------------------+-------+
-|uhv_hh                  |Alimony advance payment, household sum               | float |
-+------------------------+-----------------------------------------------------+-------+
-|kiz                     |Monthly additional child benefit, household sum      | float |
-+------------------------+-----------------------------------------------------+-------+
-|wohngeld                |Monthly housing benefit, household sum               | float |
-+------------------------+-----------------------------------------------------+-------+
-|m_alg2                  |Monthly social assistance, household sum             | float |
-+------------------------+-----------------------------------------------------+-------+
-|dpi_ind                 |Monthly disposable income, individual                | float |
-+------------------------+-----------------------------------------------------+-------+
-|dpi                     |Monthly disposable income, household sum             | float |
-+------------------------+-----------------------------------------------------+-------+
-|gross                   |Monthly market income                                | float |
-+------------------------+-----------------------------------------------------+-------+
++-------------------------+----------------------------------------------------+-------+
+|    Variable             | Explanation                                        | type  |
++=========================+====================================================+=======+
+| rentenv_beit_m          | Monthly amount employee old-age pensions contrib.  | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_v_beit_m       | Monthly amount employee unempl. insurance contrib. | float |
++-------------------------+----------------------------------------------------+-------+
+| ges_krankv_beit_m       | Monthly amount employee health insurance contrib.  | float |
++-------------------------+----------------------------------------------------+-------+
+| pflegev_beit_m          | Monthly amount of long term care insurance         | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_geld_m         | Monthly amount of unemployment assistance          | float |
++-------------------------+----------------------------------------------------+-------+
+| rente_anspr_m           | Monthly amount of old-age pensions                 | float |
++-------------------------+----------------------------------------------------+-------+
+| abgelt_st               | Monthly capital income tax due, individual         | float |
```

Comment:
All taxes should be annual, I believe:
- `abgelt_st`
- `soli_st`
- `eink_st`
- `eink_st_tu`

Taxes are just filed once per year, so it does not make much sense to divide by 12. 

Eventually, we probably want to allow for much richer ways of doing things, but that is for a much later date.

Reply:
I'd argue for keeping them in monthly terms in the output at least for now, because it is very easy to mix up quantities.

Reply:
At a minimum, we should append `_m` to the names then. No super-strong opinion here.

Reply:
Good point. I have added the `_m` to all of the taxes.